src/pudl/metadata/fields.py

```diff
@@ -2015,11 +2011,6 @@
         "unit": "USD",
     },
     "total_capacity_less_1_mw": {"type": "number", "unit": "MW"},
-    "total_cost_of_plant": {
-        "type": "number",
-        "description": "Total cost of plant (USD).",
-        "unit": "USD",
-    },
```

Comment:
Removed this because I changed the name to `capex_total`. I hope it doesn't affect any other tables.